troposphere/static/js/components/images/detail/header/HeaderView.react.js

```diff
@@ -22,6 +23,14 @@ export default React.createClass({
         });
     },
 
+    showLaunchModal: function(e) {
+        modals.InstanceModals.launch({
+            image: this.props.image,
+            initialView: "BASIC_VIEW"
+        });
+        trackAction("launched-from-image-detail", {});
```

Comment:
The `{}` might just be a carry over from the previous code and not something he introduced. 

I can speak to the fact that `trackAction` was written to allow for the second argument to be optional (supporting the expectation that `trackAction("launched-from-image-detail");` is totally *fine*).